### PR TITLE
RS-142: Update license

### PR DESCRIPTION
**Pull request by atimin**
CHanges:

* Removed mentions about BUSL
* Updated Services
* Fees exclude taxes
* Add information about plans

### CODE REVIEW

File: src/pages/terms.mdx [hunk 1]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
+1.1. **Problem** means a failure of the Supported Software to substantially conform to the functional specifications set forth in the applicable documentation provided by ReductStore.
+
+1.2. **Support Services** means each of the services made available by us under this Agreement including those offered via Architecture Review, software support and problem resolution as part of the Service offerings.
 
-## <a id="addendum-1-definitions"></a>1. Definitions
+1.3. **Updates and Upgrades** means subsequently released versions of the Supported Software. For clarity, Updates and Upgrades do not include any products that ReductStore licenses separately from the Supported Software or for an additional fee.
 
-1.1. **Problem** means a failure of the Supported Software to substantially conform to the functional specifications
-set forth in the applicable documentation provided by ReductStore.
+1.4. **Workaround** means a modification or "patch" for a particular version of the Supported Software, which may be of a temporary or interim nature, to help avoid, correct or circumvent a Problem.
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:212]: Workaround is not used anywhere else. We could stick with "Problem Resolution".

File: src/pages/terms.mdx [hunk 2]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
 
-1.4. **Workaround** means a modification or "patch" for a particular version of the Supported Software, which may be of
-a temporary or interim nature, to help avoid, correct or circumvent a Problem.
+2.2. **Maintenance**; Updates and Upgrades. As part of the Support Services, ReductStore shall use commercially reasonable efforts to maintain the Supported Software so that it operates without Problems. ReductStore shall supply Customer with Guided Updates and Upgrades for the Supported Software during the Subscription Term. Such Updates and Upgrades shall be provided to Customer at no charge.
 
-## <a id="addendum-2-support-services"></a>2. Support Services
+2.3. **Support**. Subject to Section 2.1 of this Addendum, ReductStore shall provide the Support Services described herein to assist in Customer’s use of the Supported Software and resolve Problems in the Supported Software through direct engineering support delivered remotely via the Internet. ReductStore agrees to use commercially reasonable efforts to acknowledge the Problems reported to ReductStore by Customer:
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:220]: We can use **Problem Resolution** definition here to simplify the text. For example: to assist the Customer in Problem Resolution through direct engineering support delivered remotely via the Internet.

File: src/pages/terms.mdx [hunk 3]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
 
-2.1. **Eligibility**. Upon payment of the applicable Fees, ReductStore shall provide Support Services as further
-described herein. To be eligible to receive Support Services in accordance with this Addendum.
+- in less than 2 business days after receiving such a report for Standard and IoT Plans
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:222]: Does it make sense to add a **Plan** section in  the definition? To specify Free, Standard, IoT, premium

File: src/pages/terms.mdx [hunk 4]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
+4.2.4. **Access**. The full system, including all software and hardware, is available to the Customer contact without limitations during any correspondence with ReductStore support personnel.
 
-4.2.5. **Cooperation**. The Customer contact will follow the instructions and suggestions of ReductStore’s support
-personnel when servicing the Supported Software.
+4.2.5. **Cooperation**. The Customer contact will follow the instructions and suggestions of ReductStore’s support personnel when servicing the Supported Software.
 
-4.3 **Updates and Upgrades**. Customer acknowledges and agrees that Updates and Upgrades provided by ReductStore
-pursuant
-to this Agreement may, in ReductStore’s reasonable discretion, require additional training of Customer’s personnel.
+4.3 **Updates and Upgrades**. Customer acknowledges and agrees that Updates and Upgrades provided by ReductStore pursuant to this Agreement may, in ReductStore’s reasonable discretion, require additional training of Customer’s personnel.
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:260]: 4.3. (dot)

File: src/pages/terms.mdx [hunk 5]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
 
-4.4. **Test Cases**. For certain Problems, ReductStore may require that Customer provide ReductStore a test case and
-sufficient documentation to allow duplication of the Problem and the provision of Support Services. If such test case
-and documentation are not provided, ReductStore will be relieved of the obligation to provide any Support Services
-that are dependent on such test case or documentation until Customer provides such test case or documentation.
+4.4 .**Test Cases**. For certain Problems, ReductStore may require that Customer provide ReductStore a test case and sufficient documentation to allow duplication of the Problem and the provision of Support Services. If such test case and documentation are not provided, ReductStore will be relieved of the obligation to provide any Support Services that are dependent on such test case or documentation until Customer provides such test case or documentation.
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:262]: 4.4. (space)

File: src/pages/terms.mdx [hunk 6]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
-misconfigures, alters or damages the Supported Software; (b) uses the Supported Software with any hardware or
-software not supplied or supported by ReductStore; (c) fails to install any Updates and Upgrades to the Supported
-Software if such Updates and Upgrades would have resolved the Problem; or (d) otherwise uses the Supported Software
-in a manner not in accordance with the Agreement, applicable documentation or ReductStore’s instructions.
+4.5 .**Disclaimer**. ReductStore shall not be responsible to provide Support Services, Updates and Upgrades, or any other maintenance and support to the extent that Problems arise because Customer: (a) misuses, improperly uses, misconfigures, alters or damages the Supported Software; (b) uses the Supported Software with any hardware or software not supplied or supported by ReductStore; (c) fails to install any Updates and Upgrades to the Supported Software if such Updates and Upgrades would have resolved the Problem; or (d) otherwise uses the Supported Software in a manner not in accordance with the Agreement, applicable documentation or ReductStore’s instructions.
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:264]: 4.5.

File: src/pages/terms.mdx [hunk 7]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
 
-## <a id="addendum-5-limitation-on-maintenance-and-support"></a>5. Limitation On Maintenance And Support
+## 5. Limitation On Maintenance And Support
 
-5.1. **Customer Defects**. ReductStore shall have no obligation to repair or address any Problem that is due to
-Customer’s incorrect or improper use of the Supported Software or failure to comply with the terms of this Agreement.
-However, at Customer’s request in writing approving the fees therefor, ReductStore may provide consulting services to
-correct the Problem pursuant to Section 3.3 of this Addendum.
+5.1. **Customer Defects**. ReductStore shall have no obligation to repair or address any Problem that is due to Customer’s incorrect or improper use of the Supported Software or failure to comply with the terms of this Agreement. However, at Customer’s request in writing approving the fees therefor, ReductStore may provide consulting services to correct the Problem pursuant to Section 3.3 of this Addendum.
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:268]: therefore (typo)

File: src/pages/terms.mdx [hunk 8]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
+## 2. Support Services
 
-1.3. **Updates and Upgrades** means subsequently released versions of the Supported Software. For clarity, Updates and
-Upgrades do not include any products that ReductStore licenses separately from the Supported Software or for an
-additional fee.
+2.1. **Eligibility**. Upon payment of the applicable Fees, ReductStore shall provide Support Services as further described herein. To be eligible to receive Support Services in accordance with this Addendum.
 
-1.4. **Workaround** means a modification or "patch" for a particular version of the Supported Software, which may be of
-a temporary or interim nature, to help avoid, correct or circumvent a Problem.
+2.2. **Maintenance**; Updates and Upgrades. As part of the Support Services, ReductStore shall use commercially reasonable efforts to maintain the Supported Software so that it operates without Problems. ReductStore shall supply Customer with Guided Updates and Upgrades for the Supported Software during the Subscription Term. Such Updates and Upgrades shall be provided to Customer at no charge.
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:218]: **Maintenance**. (dot)

File: src/pages/terms.mdx [hunk 9]
```diff
@@ -26,551 +18,259 @@ PLEASE READ THIS AGREEMENT CAREFULLY BEFORE PURCHASING AND/OR USING SOFTWARE OR
...
-2.1. **Eligibility**. Upon payment of the applicable Fees, ReductStore shall provide Support Services as further
-described herein. To be eligible to receive Support Services in accordance with this Addendum.
+- in less than 2 business days after receiving such a report for Standard and IoT Plans
+- in less than 4 business hours after receiving such a report for Premium Plan
 
-2.2. **Maintenance**; Updates and Upgrades. As part of the Support Services, ReductStore shall use commercially
-reasonable efforts to maintain the Supported Software so that it operates without Problems. ReductStore shall supply
-Customer with Guided Updates and Upgrades for the Supported Software during the Subscription Term. Such Updates and
-Upgrades shall be provided to Customer at no charge.
+A report initially must be sent by email to [support@reduct.store](mailto:support@reduct.store) then depending on a plan Customer can communicate:
```
Comment (AnthonyCvn) [inline src/pages/terms.mdx:225]: Link in bold

**Review by AnthonyCvn — CHANGES_REQUESTED** [review]

File: src/pages/terms.mdx (hunk 1 above)
Comment (atimin) [reply src/pages/terms.mdx:212]: Yes, but it is a part of problem resolution, so let's keep it.

File: src/pages/terms.mdx (hunk 3 above)
Comment (atimin) [reply src/pages/terms.mdx:233]: **Plan** means the level of Support Services provided by ReductStore to Customer as set forth in the Order Form.

Works?

**Review by AnthonyCvn — APPROVED** [review]
The changes look great.

**Pull request merged**